### PR TITLE
resolved for the disbaling the tabs based on services and updated code

### PR DESCRIPTION
**Pull request by Rahul139-stack**
Please review and approve it.
image

### CODE REVIEW

File: src/App.vue [hunk 1]
```diff
@@ -61,8 +63,20 @@
...
           console.log(id, output, "output")
+          if (output == "120") {
+            const experienceObj = settingsModule.data.listOfMenu.filter(
+              (element) => element.service == id
+            )
+            if (experienceObj.length > 0) {
+              experienceObj[0].disabled = false
+              experienceObj[0].icon =
+                settingsModule.data.previousIcon || experienceObj[0].icon
+              settingsModule.data.previousIcon = ""
```
Comment (Harikarthyk) [inline src/App.vue:76]: what is the previousIcon refers to ?
You can have two things activeIcon (icon) and experienceIcon (actual icon for the experience)

That should along all the objects of the listOfMenu (List of experience)

File: src/data/default.json [hunk 2]
```diff
@@ -339,6 +339,7 @@
     "settingsDialogVisible": false,
     "editDialog": false,
     "settingPath": null,
+    "previousIcon": "",
```
Comment (Harikarthyk) [inline src/data/default.json:342]: this should under settings, please check the above comment for more details.

File: src/data/default.json [hunk 3]
```diff
@@ -419,7 +420,14 @@
         "disabled": false,
         "url": "http://localhost:8113/",
         "enabled": false,
-        "subMenu": [{ "id": "load-data", "to": "#" }]
+        "subMenu": [
+          {
+            "id": "load-data",
+            "to": "#",
+            "experimental": false,
+            "icon": "pi pi-server"
```
Comment (Harikarthyk) [inline src/data/default.json:428]: have different icons for different services

**Review by Harikarthyk — CHANGES_REQUESTED** [review]

File: src/App.vue (hunk 1 above)
Comment (Rahul139-stack) [reply src/App.vue]: Done

File: src/data/default.json (hunk 2 above)
Comment (Rahul139-stack) [reply src/data/default.json:342]: Done

File: src/data/default.json (hunk 3 above)
Comment (Rahul139-stack) [reply src/data/default.json:430]: Done

File: src/components/IframeComponent/loader.vue [hunk 4]
```diff
@@ -1,128 +1,103 @@
...
-  </template>
-  <script>
-    export default {
-      name: "Loadingpage",   
-     
+  </div>
+</template>
+<script>
+  export default {
+    name: "Loadingpage",
```
Comment (Harikarthyk) [inline src/components/IframeComponent/loader.vue:22]: Can you please follow camel case for this name.

**Review by Harikarthyk — CHANGES_REQUESTED** [review]

File: src/components/IframeComponent/loader.vue (hunk 4 above)
Comment (Rahul139-stack) [reply src/components/IframeComponent/loader.vue]: Done

**Review by Harikarthyk — APPROVED** [review]

**Pull request merged**